backend/server/tests/courses/test_search_course.py

```diff
@@ -4,30 +4,55 @@
 
 import requests
 
+from server.tests.user.utility import get_token, get_token_headers
+
 with open("./algorithms/tests/exampleUsers.json", encoding="utf8") as f:
-    USER = json.load(f)["user6"]
+    USER = json.load(f)["user_degree_wizard"]
 f.close()
 
 
 def test_search_course():
-    x = requests.post('http://127.0.0.1:8000/courses/searchCourse/COMP1', json=USER)
+    user_token = get_token()
+    requests.post('http://127.0.0.1:8000/user/setupDegreeWizard', headers={
+        "Authorization": f"Bearer {user_token}"
+    }, json=USER)
+    x = requests.post('http://127.0.0.1:8000/courses/searchCourse/COMP1', headers={
+        "Authorization": f"Bearer {user_token}"
```

Comment:
after we change `get_token_headers` to take in a token for its arg, then this should not be manually built anywhere